src/resources/PowerShellWorkerStrings.resx

```diff
@@ -310,6 +310,18 @@
   <data name="CommandNotFoundException_Exception" xml:space="preserve">
     <value>CommandNotFoundException detected (exception).</value>
   </data>
+  <data name="CommandNotFoundUserWarning" xml:space="preserve">
+    <value>The app may be missing a module containing the '{0}' command definition. If this command belongs to a module available on the PowerShell Gallery, add a reference to this module to requirements.psd1. Make sure this module is compatible with PowerShell 7. For more details, see https://aka.ms/functions-powershell-managed-dependency.</value>
```

Comment:
Should we use `function app` instead of `app` for consistency? 

Also, instead of saying `PowerShell 7`, should we have a string that reference the latest version? E.g., `PowerShell {1}` where `{1}` is the current version, in this case 7.

Reply:
Agreed with "function app", fixed.
Regarding "PowerShell 7" - I'm not sure it's worth it, given the fact that the PowerShell team decided to never change the major version number, so it is highly unlikely we'll ever see "PowerShell 8" etc. If this happens, we can change this.

Reply:
Sounds good. Thanks Anatoli.